### DISCUSSION TITLE
translation/localization of tosdr [plugin]

### DISCUSSION THREAD

**Discussion by maysara** [tosdr/tosdr.org#164] [Ideas]
Greetings, I assume the messages such as "you cannot delete your account" or " Terms may be changed at any time, but you will be notified." are limited, having those strings translated greatly increases the usefulness of tosdr! I could not find useful language variables, i18n, l10n, translation, .po files ...etc, I think it is worth introducing ...

**Comment by michielbdejong** [comment]
good point! one problem is that most services will provide localized ToS and so a localized review should also be about those localized terms. But that's doable. I think the main reason we haven't done this yet is that we haven't had time yet to work on translating all these hundreds of strings into multiple languages.
